src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs

```diff
@@ -110,11 +110,13 @@ public static class LogEventExtensions
         {
             if (logEvent == null) throw new ArgumentNullException("logEvent");
             if (logEvent.Exception == null) throw new ArgumentException("Must have an Exception", "logEvent");
-            
+
             var exceptionTelemetry = new ExceptionTelemetry(logEvent.Exception)
             {
                 SeverityLevel = logEvent.Level.ToSeverityLevel(),
+#if !NETSTANDARD1_6
```

Comment:
Sure thing, out of interest is this just due to the possibity of breaking things if new frameworks are added?